### PR TITLE
feat: use http and kyt

### PR DESCRIPTION
**Pull request by martonmoro**
Modify task to mock a KYT

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
-# Incredible Squaring AVS
+# EigenLayer KYT AVS
```
Comment (gabkov) [inline README.md]: add KRNL pls

File: aggregator/aggregator.go [hunk 2]
```diff
@@ -193,14 +200,30 @@ func (agg *Aggregator) sendAggregatedResponseToContract(blsAggServiceResp blsagg
...
+	var address AddressToKYT
+	err := json.NewDecoder(r.Body).Decode(&address)
+	// numToSquare := req.URL.Query().Get("numToSquare")
+	if err != nil {
+		// Handle error
+	}
+	if address.Address != "" {
+		err := agg.sendNewTaskKYT(&address.Address)
+		if err != nil {
+			// we log the errors inside sendNewTask() so here we just continue to the next task
```
Comment (gabkov) [inline aggregator/aggregator.go]: log the error here

File: config-files/aggregator-docker-compose.yaml
Comment (gabkov) [inline config-files/aggregator-docker-compose.yaml:1]: delete porbably

File: config-files/operator-docker-compose.anvil.yaml
Comment (gabkov) [inline config-files/operator-docker-compose.anvil.yaml:1]: delete probably

File: contracts/bindings/IncredibleSquaringTaskManager/binding.go
Comment (gabkov) [inline contracts/bindings/IncredibleSquaringTaskManager/binding.go:1]: yeet

File: contracts/src/KYTTaskManager.sol [hunk 3]
```diff
@@ -43,6 +43,9 @@ contract IncredibleSquaringTaskManager is
 
     mapping(uint32 => bool) public taskSuccesfullyChallenged;
 
+    //TODO: change to address
```
Comment (gabkov) [inline contracts/src/KYTTaskManager.sol]: yeet

File: core/chainio/mocks/avs_reader.go
Comment (gabkov) [inline core/chainio/mocks/avs_reader.go:1]: yeet mocks if not needed

File: operator/operator.go [hunk 4]
```diff
@@ -36,6 +35,17 @@ import (
 const AVS_NAME = "incredible-squaring"
```
Comment (gabkov) [inline operator/operator.go]: rename

File: operator/operator.go [hunk 5]
```diff
@@ -36,6 +35,17 @@ import (
...
 const SEM_VER = "0.0.1"
 
+const (
```
Comment (gabkov) [inline operator/operator.go:39]: comment: simulated verified db or similar

File: operator/operator.go [hunk 6]
```diff
@@ -303,24 +313,32 @@ func (o *Operator) Start(ctx context.Context) error {
...
-		"numberToBeSquared", newTaskCreatedLog.Task.NumberToBeSquared,
+		"addressToBeKYTd", newTaskCreatedLog.Task.AddressToKYT,
 		"taskIndex", newTaskCreatedLog.TaskIndex,
 		"taskCreatedBlock", newTaskCreatedLog.Task.TaskCreatedBlock,
 		"quorumNumbers", newTaskCreatedLog.Task.QuorumNumbers,
 		"QuorumThresholdPercentage", newTaskCreatedLog.Task.QuorumThresholdPercentage,
 	)
-	numberSquared := big.NewInt(0).Exp(newTaskCreatedLog.Task.NumberToBeSquared, big.NewInt(2), nil)
-	taskResponse := &cstaskmanager.IIncredibleSquaringTaskManagerTaskResponse{
+	// TODO: implement KYT
```
Comment (gabkov) [inline operator/operator.go]: todo?

File: tests/anvil/deploy-avs-save-anvil-state.sh [hunk 7]
```diff
@@ -1,27 +1,28 @@
...
 parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 cd "$parent_path"
 
 # start an anvil instance in the background that has eigenlayer contracts deployed
-anvil --load-state eigenlayer-deployed-anvil-state.json --dump-state avs-and-eigenlayer-deployed-anvil-state.json &
+anvil --port 8546 --load-state eigenlayer-deployed-anvil-state.json --dump-state avs-and-eigenlayer-deployed-anvil-state.json &
 cd ../../contracts
-forge script script/IncredibleSquaringDeployer.s.sol --rpc-url $RPC_URL --private-key $PRIVATE_KEY --broadcast -v
+forge script script/KYTDeployer.s.sol --rpc-url $RPC_URL --private-key $PRIVATE_KEY --broadcast -v
+# forge script script/IncredibleSquaringDeployer.s.sol --rpc-url $RPC_URL --private-key $PRIVATE_KEY --broadcast -v
```
Comment (gabkov) [inline tests/anvil/deploy-avs-save-anvil-state.sh]: yeet

File: tests/integration/integration_test.go
Comment (gabkov) [inline tests/integration/integration_test.go:1]: yeetable?

Comment (martonmoro) [reply tests/integration/integration_test.go:1]: I believe we can keep this

**Review by gabkov — APPROVED** [review]
lgtm

**Pull request merged**